### ISSUE TITLE
Doesn't appear to handle multiple cover profiles concatenated

### ISSUE THREAD

**Issue by foresmac** [vokal/golang-cover-parse#2]
This is what I talking about with the multiple `mode: count` lines. The coverage for vip has info for other files that aren't included as well, but isn't listed in cvr.
- `fetch/fetch.go`
- `fetch/mainip.go`
- `fetch/context.go`

**Comment by jrit** [comment]
Will take a look at it

**Comment by foresmac** [comment]
The other trick here is that files that aren't hit at all don't even show up in the profile. There are other files in the project but aerent counted _against_ the coverage even though the probably should be. I don't know if that means we should do more to generate a better coverage report or if we need to do more to look at the whole source of project.

**Comment by foresmac** [comment]
Or a little of both.

**Comment by foresmac** [comment]
**TL;DR** Ignore the part about counting files that don't show up in the cover profile; that's on our team to fix with better standards about writing tests for Go projects.

I'm going to investigate what alternatives are available for generating coverage reports, or maybe draft some standards for writing tests for Go projects, because the cover profile generated by `go test` will only ever refer to files in the package which they are linked to. For instance, even though we have a full suite of resizing tests, that test is part of `package main` and therefore doesn't record any coverage for `fetch/mainip.go` where all the resizing code lives.
